src/main/java/com/test/Test.java

```diff
@@ -0,0 +1,9 @@
+package com.test;
+
+public class Test {
+	public static void main(String[] args) {
+		System.out.println("test class");
+		System.out.println("once check it ");

```

Comment:
insted of main method create your own method

Reply:
ok